src/api/singa.i

```diff
@@ -25,10 +25,5 @@
 %include "config.i"
 %include "core_tensor.i"
 %include "core_device.i"
-%include "model_layer.i"
-%include "model_optimizer.i"
```

Comment:
Without `%include "model_optimizer.i"` the old optimizer test case does not work

```
======================================================================
ERROR: test_sgd (test_optimizer.TestOptimizer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/dcsysh/singa/test/python/test_optimizer.py", line 77, in test_sgd
    sgd = opt.SGD(lr)
  File "/root/dcsysh/singa/build/python/singa/optimizer.py", line 222, in __init__
    self.opt = singa.CreateOptimizer('SGD'.encode())
AttributeError: module 'singa.singa_wrap' has no attribute 'CreateOptimizer'
```

Reply:
But even if I put the `%include "model_optimizer.i"` back, there is another error
```
======================================================================
ERROR: test_sgd_cuda (test_optimizer.TestOptimizer)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/root/dcsysh/singa/test/python/test_optimizer.py", line 117, in test_sgd_cuda
    sgd = opt.SGD(lr)
  File "/root/dcsysh/singa/build/python/singa/optimizer.py", line 222, in __init__
    self.opt = singa.CreateOptimizer('SGD'.encode())
TypeError: in method 'CreateOptimizer', argument 1 of type 'std::string const &'
```

The c optimizer is very old and decreprated (not used anymore)? Our current optimizer is in python